src/components/editor/Controls.tsx

```diff
@@ -66,13 +100,12 @@ export default function Controls({ exportHandler, exporting }: {
 				</Switch.Root>
 			</Label>
 		</div>
+		<p>Estimated GIF Size: {estimatedSize}mb </p>
```

Comment:
Nit: "Mb"? or is "mb" intentional?

Reply:
Should be Mb. Will take care of this in UI design PR.